src/test/java/filehandler/FileHandlerTest.java

```diff
@@ -0,0 +1,127 @@
+package filehandler;
+
+import exceptions.SEPEmptyException;
+import exceptions.SEPException;
+import exceptions.SEPUnknownException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import parser.Parser;
+import student.Student;
+import studentlist.StudentList;
+import ui.UI;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class FileHandlerTest {
+    private FileHandler fileHandler;
+    private Parser parser;
+    private StudentList studentList;
+    private UI ui;
+    private ArrayList<Student> results;
+
+    private static final Path CSV_FILE_PATH = Paths.get("test.csv");
```

Comment:
The CI tests are failing as `checkstyleMain` requires the static variable declaration first 😅